inbox/models/message.py

```diff
@@ -306,6 +306,27 @@ def create_from_synced(cls, account, mid, folder_name, received_date,
                     setattr(msg, field, [])
                     msg._mark_error()
 
+        self.subject = self.subject if self.subject else ''
+        self.snippet = self.snippet if self.snippet else ''
+        self.body = self.body if self.body else ''
+
+        batch_input = [
+            {"plaintext": self.subject},
+            {"plaintext": self.snippet},
+            {"plaintext": self.body},
+            {"plaintext": body_string},
+        ]
+
+        named_key = 'account-' + account.namespace.public_id
```

Comment:
is the public_id = account_id in that case?

Reply:
yes. It's the same **public_id** that we use in Dexter and Orhestra

Reply:
I've tested locally: nylas encrypts, encrypted data is passed to Orchestra mails, Orchestra mail processes it and saves it enctypted in MongoDB. And user gets unencrypted data on frontend